src/test/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipControllerTest.java

```diff
@@ -48,7 +48,6 @@ void init() {
         limitedPartnershipSubmissionDto = new LimitedPartnershipSubmissionDto();
         limitedPartnershipSubmissionDto.setData(data);
     }
-
```

Comment:
Why has this empty line be removed? I would just revert the changes to this file.